apps/prairielearn/src/lib/base64-util.ts

```diff
@@ -1,10 +1,9 @@
-// @ts-check
 import { logger } from '@prairielearn/logger';
 
-const atob = (s) => String.fromCharCode(...Buffer.from(s, 'base64'));
-const btoa = (s) => Buffer.from(s.split('').map((c) => c.charCodeAt(0))).toString('base64');
+const atob = (s: string) => String.fromCharCode(...Buffer.from(s, 'base64'));
+const btoa = (s: string) => Buffer.from(s.split('').map((c) => c.charCodeAt(0))).toString('base64');
 
-export function b64EncodeUnicode(str) {
+export function b64EncodeUnicode(str: string | number | boolean) {
```

Comment:
Nit, do we actually want to allow `number` and `boolean` here? I know `encodeURIComponent` accepts them, but it'd feel like a misuse of this function to me, especially since one can't losslessly round trip them through encoding and decoding.

Reply:
seems reasonable.